src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -211,13 +212,14 @@ public final class OpenSearchSecurityPlugin extends OpenSearchSecuritySSLPlugin
     private volatile ThreadPool threadPool;
     private volatile ConfigurationRepository cr;
     private volatile AdminDNs adminDns;
-    private volatile ClusterService cs;
+    private static volatile ClusterService cs;
```

Comment:
`static` has caused issues with testing where if a testCluster is spun up with 3 nodes in the same JVM, only one of the ClusterServices wins and 2 ClusterServices are not available.

I'd like to look into alternatives to declaring this static.

Reply:
Since we only need the clusterID, I did a work around by doing [this](https://github.com/opensearch-project/security/pull/3179/commits/e8ef7ea81dbdea2cc3d0135f78aba75f482b67ae).